docs/utils/iers.rst

```diff
@@ -23,9 +23,13 @@ transformations.
 Getting started
 ===============
 
-Starting with astropy 1.2, the latest IERS values (which include approximately
```

Comment:
This behavior has not changed and so the changes from here through line 32 should be reverted.

Reply:
Reverted.